### ISSUE TITLE
php role breaking ansible-galaxy install

### ISSUE THREAD

**Issue by robdecker** [geerlingguy/ansible-role-php#110]
When installing the `ansible-galaxy` I'm getting an error that is breaking the install:

`$ sudo ansible-galaxy install -r provisioning/requirements.yml --force`

```
- downloading role 'php', owned by geerlingguy
https://galaxy.ansible.com/roles/432/versions/?page=2&page_size=50
Traceback (most recent call last):
  File "/usr/local/bin/ansible-galaxy", line 957, in <module>
    main()
  File "/usr/local/bin/ansible-galaxy", line 951, in main
    fn(args, options, parser)
  File "/usr/local/bin/ansible-galaxy", line 819, in execute_install
    if len(role_versions) > 0:
TypeError: object of type 'NoneType' has no len()
```

**Comment by oxyc** [comment]
This is an upstream bug in galaxy. It's fixed in Ansible 2, but the fix has not been released in 1.9.x yet. See https://github.com/ansible/galaxy-issues/issues/134

**Comment by robdecker** [comment]
Upgrading Ansible took care of it. Much thanks!
